composer.json

```
@@ -17,7 +17,7 @@
         "flix-tech/confluent-schema-registry-api": "^6.0",
         "symfony/console": "^4.3",
         "pimple/pimple": "^3.2",
-        "jobcloud/php-kafka-schema-registry-client": "^0.1",
+        "jobcloud/php-kafka-schema-registry-client": "dev-feat/JMP-2237/registry-import-modes@dev",
```

```suggestion
        "jobcloud/php-kafka-schema-registry-client": "^0.1",
```
you can revert this change back, release was created